### PR TITLE
Subscription price change

### PR DESCRIPTION
**Pull request by RafaelHiguera**
Adding subscription price change pop up when the user opens the store.

### CODE REVIEW

**Review by searoadtosoftware — APPROVED** [review]

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs [hunk 1]
```diff
@@ -319,28 +319,33 @@ private static void UnlockInGameContent(string productId)
...
+        // This situation may happen when the user lose internet connection after paying to play store.
         if (failureReason == PurchaseFailureReason.DuplicateTransaction)
         {
             _playStoreExtensions.RestoreTransactions(null);
         }
     }
 
+    public static void confirmSubscriptionPriceChange(string productId)
+    {
+        _playStoreExtensions.ConfirmSubscriptionPriceChange(productId, delegate(bool success) { });
```
Comment (Qing451800) [inline GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs]: You need to implement this call back. For the simplest user case, use Debug.log to print it out and add comment saying that devs can choose to make update or record that user has accepted the new price.

**Review by Qing451800 — CHANGES_REQUESTED** [review]

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs (hunk 1 above)
Comment (RafaelHiguera) [reply GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs]: I see, will do 👍

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs [hunk 2]
```diff
@@ -319,28 +318,45 @@ private static void UnlockInGameContent(string productId)
...
+            delegate(bool priceChangeSucess)
+            {
+                if (priceChangeSucess)
+                {
+                    // Here you can choose to make an update or record that the user accpected the new price
+                    Debug.Log("The user accepted the price change");
+                }
+                else
+                {
+                    Debug.Log("The did not accecpt the price change");
```
Comment (Qing451800) [inline GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs]: The user did not accecpt the price change

**Review by Qing451800 — APPROVED** [review]

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs (hunk 2 above)
Comment (RafaelHiguera) [reply GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs]: Oops, going to fix it

**Pull request merged**